### ISSUE TITLE
v0.5.0 release/roadmap

### ISSUE THREAD

**Issue by Kukkimonsuta** [ui-router/react#42]
Is there anything I can do to speed up release of build compatible with ui-router-core 5.0.0?

**Comment by Schlesiger** [comment]
I also have a project eagerly awaiting these features. If you'd like some testers, I'll have time in the near future to pull this branch and give it a try.

**Comment by elboman** [comment]
Hi,
Actually 0.5.0 is pretty much done, I was going to update the docs in the source but I guess we could release the package on npm and work on the docs later on.
Have you tried the branch out yet? Did you find any issue?

**Comment by Kukkimonsuta** [comment]
I've tried upgrading to core `5.0.0` and example seem to work fine, however tests for `UISref` are failing 

https://github.com/Kukkimonsuta/react/tree/switch-to-uirouter-5

If I disable the empty `defaultErrorHandler` I get:
```
    console.error ..\node_modules\@uirouter\core\lib\state\stateService.js:40
      Transition Rejection($id: 9 type: 6, message: The transition errored, detail: TypeError: Cannot read property 'reduce' of undefined)
    console.error ..\node_modules\@uirouter\core\lib\state\stateService.js:42
      TypeError: Cannot read property 'reduce' of undefined
          at ResolveContext.Object.<anonymous>.ResolveContext.resolvePath (D:\Projects\ui-router-react\node_modules\@uirouter\core\src\resolve\resolveCo
ntext.ts:132:46)
          at lazyResolveState (D:\Projects\ui-router-react\node_modules\@uirouter\core\src\hooks\resolve.ts:40:10)
          at invokeCallback (D:\Projects\ui-router-react\node_modules\@uirouter\core\src\transition\transitionHook.ts:93:23)
          at TransitionHook.Object.<anonymous>.TransitionHook.invokeHook (D:\Projects\ui-router-react\node_modules\@uirouter\core\src\transition\transit
ionHook.ts:105:20)
          at D:\Projects\ui-router-react\node_modules\@uirouter\core\src\transition\transitionHook.ts:208:34
          at process._tickCallback (internal/process/next_tick.js:109:7)
```

**Comment by Schlesiger** [comment]
After some refactoring, it's working well for me! I have not tested `<UISref />`'s, though. Also, will `0.5.0` be upgraded to use `@uirouter/core`? I'm curious if issues [44](https://github.com/ui-router/core/issues/44) and [45](https://github.com/ui-router/core/issues/45) are resolved with the lastest. Thanks!

**Comment by Schlesiger** [comment]
I've created a fork and checked in the `lib` and `lib/esm` directories. If anyone else wants to try the `0.5.0` branch before it's published, you can replace the following in your `settings.json`:
```json
"ui-router-react": "git+https://github.com/Schlesiger/react.git#0.5.0"
```

**Comment by elboman** [comment]
I just released `0.5.0` under the new org scoped name `@uirouter/react`.

Feedback and contributions always welcome!
